### ISSUE TITLE
Add content to draft runbook outline of a11y/QA portion for Drupal upgrades

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15857]
## Description or Additional Context 
I created [an outline](https://vfs.atlassian.net/wiki/spaces/PCMS/pages/2804711432/Runbook+Draft+-+Accessibility+QA+Testing) for the accessibility/QA portion of the Drupal upgrade runbook in #15433.

This issue is intended to cover the work of fleshing this out into a more descriptive, more informative draft.

## Acceptance Criteria
- [x] Accessibility-specific portion of the runbook has been drafted.
- [x] QA-specific portion of the runbook has been drafted.
- [x] Product has reviewed the draft and approved it.
- [x] Draft has been saved in Confluence.

## Team
Please check the team(s) that will do this work.

- [x] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

**Comment by laflannery** [comment]
@EWashb and @BerniXiongA6 I completed [the accessibility part](https://vfs.atlassian.net/wiki/spaces/PCMS/pages/2804711432/Accessibility+Testing+for+Drupal+Upgrades) of this. When you are able can you review and let me know if you have any feedback/edits?

**Comment by EWashb** [comment]
@laflannery this looks good. I like that you linked out to additional resources. I think it is important to note where our standards are and how they align with additional standards we use for accessibility. Feel free to link out to additional resources and guidance as you see fit.

**Comment by laflannery** [comment]
@EWashb Good catch on the fact that I didn't mention accessibility standards at all, I [added that to the top](https://vfs.atlassian.net/wiki/spaces/PCMS/pages/2804711432/Accessibility+Testing+for+Drupal+Upgrades#Accessibility-standards)

I also added a [draft for the QA portion](https://vfs.atlassian.net/wiki/spaces/PCMS/pages/2827125110/QA+Testing+for+Drupal+Upgrades) but I have a question:
- In the "What to test" section, you will notice that is has a list of everything which is because I have ended up in this default position of doing QA and I just review everything while I'm doing the accessibility testing. However do we want this list to more accurately reflect what the CMS team owns and therefore what the CMS team would actually be responsible for QA-ing?

Let me know what you think and also if there are other edits/updates either of you think I should make @EWashb & @BerniXiongA6

**Comment by EWashb** [comment]
The updates on the accessibility standards looks great! And thank you for going in to do the QA stuff too. I wonder if @JunTaoLuo would be interested in reviewing it too? Or maybe even @BlakeOrgan from the UX/QA pov?

**Comment by laflannery** [comment]
I would love their feedback, especially @JunTaoLuo because he is here now and I'm sure knows so much more about how this should be done than I do

**Comment by JunTaoLuo** [comment]
Thanks for pinging me on this as it's something I want to keep an eye on. I agree with what's in the outline saying should rely on automated testing (both for a11y and functionality) as much as possible but also conduct manual testing for the main user workflows/stories to make sure we don't introduce any major blockers. In the future, we're planning to add more to the automated cypress tests but I don't think we have that work fully refined yet.

As for the specific details of what we need to test, I'll need to rope in @edmund-dunn and @ndouglas since they have a much better understanding of all the CMS components and experience with previous Drupal upgrades. cc https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16047 since it might be related.

**Comment by laflannery** [comment]
Thanks @JunTaoLuo! 

With all that in mind, I think I'm going to go ahead an close this particular ticket for now since the goal was just to get a draft in Confluence and have interested parties take a first pass and make sure things were on the right track. Also, since the a11y and QA were grouped together for reasons, now I think we should move this over to additional, separate tickets for each practice area and they can be reviewed more in depth and finalized by the correct people.

CC: @EWashb and @BerniXiongA6
